paasta_tools/cli/cmds/validate.py

```diff
@@ -715,6 +728,68 @@ def validate_secrets(service_path):
     return return_value
 
 
+def validate_cpu_burst(service_path: str) -> bool:
+    soa_dir, service = path_to_soa_dir_service(service_path)
+    skip_cpu_burst_validation_list = (
+        load_system_paasta_config().get_skip_cpu_burst_validation_services()
+    )
+
+    returncode = True
+    for cluster in list_clusters(service, soa_dir):
+        for instance in list_all_instances_for_service(
+            service=service, clusters=[cluster], soa_dir=soa_dir
+        ):
+            instance_config = get_instance_config(
+                service=service,
+                instance=instance,
+                cluster=cluster,
+                load_deployments=False,
+                soa_dir=soa_dir,
+            )
+            is_k8s_service = instance_config.get_instance_type() == "kubernetes"
+            should_skip_cpu_burst_validation = service in skip_cpu_burst_validation_list
+            if is_k8s_service and not should_skip_cpu_burst_validation:
+                # we need access to the comments, so we need to read the config with ruamel to be able
+                # to actually get them in a "nice" automated fashion
+                config = get_config_file_dict(
+                    os.path.join(soa_dir, service, f"kubernetes-{cluster}.yaml"),
+                    use_ruamel=True,
+                )
+
+                if config[instance].get("cpu_burst_add") is None:
+                    # using autotuned values - can skip
+                    continue
+                if config[instance]["cpu_burst_add"] <= CPU_BURST_THRESHOLD:
+                    # under the threshold - can also skip
+                    continue
+
+                burst_comment = _get_comments_for_key(
+                    data=config[instance], key="cpu_burst_add"
+                )
+                # we could probably have a separate error message if there's a comment that doesn't match
+                # the ack pattern, but that seems like overkill - especially for something that could cause
+                # a DAR if people aren't being careful.
+                if (
+                    burst_comment is None
+                    or re.search(
+                        pattern=OVERRIDE_CPU_BURST_ACK_PATTERN,
+                        string=burst_comment,
+                    )
+                    is None
+                ):
+                    returncode = False
+                    print(
+                        failure(
+                            msg=f"Potentially excessive CPU burst (cpu_burst_add: {config[instance]['cpu_burst_add']} "
+                            f"higher than current threshold of {CPU_BURST_THRESHOLD} cores) detected in {cluster}: {service}.{instance}."
+                            " Please read the following link for next steps:",
```

Comment:
might be nice to print `OVERRIDE_CPU_BURST_ACK_PATTERN` in this message, so people who have seen this error before can quickly resolve it without reading docs

Reply:
yea, I guess my train of thought here was that if you've seen this before and know what you're doing you'll be able to quickly scan the docs to get the right string OR you'll remember the previous occurrence of this and grab the string from there - imo, it's kinda nice to have a little bit of friction here to prevent folks from setting too high a value carelessly